vite.config.ts

```diff
@@ -80,7 +88,7 @@ export default defineConfig({
     // https://github.com/antfu/vite-plugin-pwa
     VitePWA({
       registerType: 'autoUpdate',
-      includeAssets: ['favicon.svg', 'robots.txt', 'safari-pinned-tab.svg'],
+      includeAssets: ['favicon.svg', 'sitemap.xml', 'robots.txt', 'safari-pinned-tab.svg'],
```

Comment:
should be included on the sw precache manifest? this file should be only used by crawlers, do you have some link about this?

Reply:
I'm not sure to understand your question.. i just thinked that sitemap.xml can be read by users, and like robots.txt, this is a needed add.

Reply:
`sitemap.xml` is primaly used by `crawlers` to allow search in your site (indexing process for SEO), the browser will not show anything related to the `sitemap.xml` file, you can install some extension to explore it (https://chrome.google.com/webstore/detail/sitemap-explorer/jamphegminpokpnalkjiecfoobdnlmfb).

In fact we should ping crawlers for new deployments: of course anyone can read the `sitemap.xml` and add the sitemap entry to their app. The crawlers will not use the sw and so my question.

you can check this docs:
- https://developers.google.com/search/docs/advanced/sitemaps/overview
- https://developers.google.com/search/docs/advanced/sitemaps/build-sitemap

Reply:
If the browser do not show anything related to the sitemap, do `robots.txt` have to be removed from VitePWA includeAssets?

Reply:
good question

Reply:
You are right, i have removed `sitemap.xml` and `robots.txt` in commit ebee2d81f9ad4836b2d13df6c47337e5ebf4c21a

Reply:
Check if offline works, chrome shouln't complain

Reply:
No problem in offline mode on my side